src/main.rs

```diff
@@ -28,8 +28,7 @@ fn main() {
 
     match matches.subcommand() {
         ("cli", Some(cli_matches)) => match cli_matches.subcommand() {
-            ("sign", Some(sign_matches)) => cli::sign(&setup, sign_matches),
-            ("type_hash", Some(type_hash_matches)) => cli::type_hash(&setup, type_hash_matches),
+            ("type_hash", _) => cli::type_hash(&setup),
             ("keygen", _) => cli::keygen(),
             _ => unreachable!(),
```

Comment:
I've thought about this and I decided to keep them there for the following reasons:

1. `type_hash` here can be used to fetch type hash for the `always_success` script. Even though this will be much less likely to change, it will be quite handy when something in genesis block changes and we want to find out the new type hash for using `always_success` script
2. `keygen` provides a way to generate keys when we don't want to deal with installing secp256k1 Ruby gems, at least until [this](https://github.com/cryptape/ruby-bitcoin-secp256k1/issues/6) is fixed

Thoughts?

Reply:
fine, so be it